business/dashboards.go

```diff
@@ -103,6 +107,14 @@ var istioCharts = []istioChart{
 		},
 		refName: "request_duration",
 	},
+	{
+		Chart: kmodel.Chart{
+			Name:  "Request duration",
+			Unit:  "ms",
```

Comment:
I believe this unit is not going to scale properly when displayed on charts if it has big or low values (e.g. 1000 ms might be displayed as "1 kms" for "kilo-millisecond" :man_facepalming: 

Have you seen something like that? (I haven't enabled telemetry lite so I can't confirm right now)

This is not something you can fix in this PR, I would need to do something in k-charted.